package.json

```diff
@@ -124,6 +124,9 @@
       "\\.js$": "jest-esm-jsx-transform",
       "^.+\\.tsx?$": "ts-jest"
     },
+    "setupFilesAfterEnv": [
+      "./tests/setup.js"
```

Comment:
https://github.com/facebook/jest/issues/2582 and it's not resolved (only workarounds)